config.json.example

```diff
@@ -2,87 +2,111 @@
   "accounts": [
     {
       "auth_service": "google",
-      "username": "test@gmail.com",
-      "password": "password",
-      "location": "University of Southern California, Los Angeles, CA",
+      "username": "@gmail.com",
+      "password": "",
+      "location": "32.714627, -117.162521",
```

Comment:
Yea I'll make it pretty asap, just wanted to push what I had and sleep for the night.